BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -0,0 +1,377 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "BSP_CAN.h"
+#include "stm32f4xx.h"
+#include "os.h"
+
+// The message information that we care to receive
+typedef struct _msg {
+    uint32_t id;
+    uint8_t data[8];
+} msg_t;
+
+// Set up a fifo for receiving
+#define FIFO_TYPE msg_t
+#define FIFO_SIZE 10
+#define FIFO_NAME msg_queue
+#include "fifo.h"
+
+static msg_queue_t gRxQueue;
```

Comment:
Why do we just have one global rx queue now? How are we to differentiate between the two busses?